### PR TITLE
console - add the possibility to disable recaptcha at account creation

### PR DESCRIPTION
**Pull request by pmauduit**
By default the recaptcha integration is activated, but adding:

```
recaptcha.activated=false
```

Allows to deactivate the recaptcha integration.

Tests: runtime on previous flag set to false / true. With false, I was able to create my account without having to fill the recaptcha field.

### CODE REVIEW

**Comment by landryb** [comment]
\o/ \o/ thanks @pmauduit - will make sure to test that shortly.

**Comment by cmangeat** [comment]
(https://github.com/georchestra/georchestra/blob/master/console/src/main/webapp/WEB-INF/views/passwordRecoveryForm.jsp#L40)

**Comment by landryb** [comment]
https://github.com/georchestra/georchestra/blob/master/console/src/main/webapp/WEB-INF/views/passwordRecoveryForm.jsp#L74 too :)

**Comment by pmauduit** [comment]
hmmm seems that it is still in a WIP state ... sorry for the points I missed out.

**Comment by pmauduit** [comment]
Ok reworked using the following strategy:

1. grep 'recaptcha/api.js' in every jsp files, got 'createAccountForm' and 'passwordRecoveryForm'.
2. grep for the 2 previous view names into every Java source files, got 'EditUserDetailsFormController.java' and 'NewAccountFormController.java' from the first one, "NewPasswordFormController.java' and 'PasswordRecoveryFormController.java' from the second one.
3. Ensure that everything is disabled (if / when statements in java / jsp code)
4. Last pass git grepping onto `RecaptchaUtils.validate()`, ensuring that there were conditional statements around.

Untested at runtime after the 2nd commit, @landryb care to have a look ?

File: console/src/main/webapp/WEB-INF/views/passwordRecoveryForm.jsp [hunk 1]
```diff
@@ -90,7 +94,12 @@
 	<script type="text/javascript">
     /* Validate the form */
     function validate() {
+<c:when test="${recaptchaActivated}">
```
Comment (landryb) [inline console/src/main/webapp/WEB-INF/views/passwordRecoveryForm.jsp]: needs to be wrapped within `<c:choose>` & </c:choose>`

**Comment by landryb** [comment]
Password recovery page blows:
```
** Root cause is: Illegal use of <when>-style tag without <choose> as its direct parent
javax.servlet.jsp.JspTagException: Illegal use of <when>-style tag without <choose> as its direct parent
```

then when/otherwise bits just needs to be wrapped within `<c:choose>` tags. With that fixed, account creation & password recovery pages work fine with recaptcha disabled.

File: console/src/main/webapp/WEB-INF/views/passwordRecoveryForm.jsp (hunk 1 above)
Comment (pmauduit) [reply console/src/main/webapp/WEB-INF/views/passwordRecoveryForm.jsp]: Wooops sorry ...

**Comment by fvanderbiest** [comment]
LGTM but missing var in https://github.com/georchestra/datadir/blob/master/console/console.properties

**Comment by pmauduit** [comment]
@fvanderbiest  https://github.com/georchestra/datadir/pull/94

Good to go ?

**Comment by fvanderbiest** [comment]
LGTM, you can merge, unless you expect someone else's review.

**Pull request merged**